heat_and_trees/heat_and_trees.ipynb

```diff
@@ -595,8 +626,7 @@
     "thresholded_temp_p = (mean_temp_f.where(mean_temp_f > 90)\n",
```

Comment:
I find the formatting of this cell a bit difficult to parse, could you try to make it clearer?

Reply:
Yes. Will do.

Reply:
Here's my suggested formatting for this cell:
```python
thresholded_temp = mean_temp_f.where(mean_temp_f > 90)
thresholded_temp_p = thresholded_temp.hvplot(
    x='x', y='y', title='Mean Temp (F) > 90', cmap=special_cmap,
    crs=crs, frame_width=400, frame_height=450, legend=False
)
thresholded_temp_p + thresholded_temp_p.opts(alpha=.3, data_aspect=None) * EsriImagery
```